CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 [Unreleased][Unreleased_log]
 --------------
+### Added
+- Enabled creation of enclaves with base address 0x0 in SGX on Linux. In these enclaves a page fault is thrown on NULL pointer dereference. This enables applications to adopt NullPointerException/ NullReferenceException in their application or use other programming stacks that do (Example, .NET runtime). This feature requires the PSW to be updated.
+  - Developers can create an enclave with base address 0x0 by setting the oesign tool configuration option 'CreateZeroBaseEnclave' to 1 or by passing in argument CREATE_ZERO_BASE_ENCLAVE=1 in OE_SET_ENCLAVE_SGX().
```

Comment:
It would be nicer to create a line item for zero-based enclaves and mention these as nested sub-points.